### PR TITLE
`Development`: Optimize and sort imports for e2e tests

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Currently the imports of the e2e tests are messy, not sorted and not consistent. 

### Description

This PR optimizes, sorts and makes the imports within the e2e tests consistent. 

#### Code Review
- [ ] Review 1
- [ ] Review 2

### CODE REVIEW

File: src/test/cypress/e2e/Logout.cy.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { Course } from '../../../main/webapp/app/entities/course.model';
-import { ModelingExercise } from '../../../main/webapp/app/entities/modeling-exercise.model';
+import { Course } from 'app/entities/course.model';
+import { ModelingExercise } from 'app/entities/modeling-exercise.model';
+
```
Comment (Strohgelaender) [inline src/test/cypress/e2e/Logout.cy.ts:3]: are these empty lines on purpose?

**Review by JohannesStoehr — COMMENTED** [review]
We had a similar PR for the client imports here: #5760
This also introduced a rule in the linter that checks the imports, can we do something similar here?

File: src/test/cypress/e2e/Logout.cy.ts (hunk 1 above)
Comment (TheZoker) [reply src/test/cypress/e2e/Logout.cy.ts:3]: Yes these empty lines are on purpose to separate the following types of imports:
1. Imports from external dependencies (e.g. `dayjs`)
2. Imports from the client code
3. Imports from within the e2e environment

**Comment by TheZoker** [comment]
@JohannesStoehr Thanks for the review. Actually these rules are already also applied to the e2e tests, but they only cover the sorting within one import element. They do not sort the imports overall. I looked a little bit into a linting rule for import sorting of that kind and found this:
https://github.com/trivago/prettier-plugin-sort-imports

I'm not sure if this is something that is wanted for the whole project. If yes I can certainly look into adding linting rules using that plugin.
For now I just used this VSCode extension with a custom config:
https://marketplace.visualstudio.com/items?itemName=mike-co.import-sorter

**Comment by Strohgelaender** [comment]
@TheZoker In this PR my initial goal was to avoid imports of this form:
```
import { a } from 'x';
import { b } from 'x';
```
and automatically merge them to:
```
import { a, b } from 'x';
```
which was solved using the import sorting rule. 

However, I think we can also activate import sorting for the whole project. Should I add this directly to this PR?

**Pull request closed without merging**